test/test_torch.py

```diff
@@ -8053,28 +8053,30 @@ def test_complex_rot90(self, device, dtype):
             np_fn = partial(np.rot90, k=rot_times, axes=[0, 1])
             self.compare_with_numpy(torch_fn, np_fn, data)
 
+    @unittest.skipIf(not TEST_SCIPY, "Scipy not found")
     def test_signal_window_functions(self, device):
-        if not TEST_SCIPY:
-            raise unittest.SkipTest('Scipy not found')
 
-        def test(name):
+        def test(name, args=()):
             torch_method = getattr(torch, name + '_window')
             for size in [1, 2, 5, 10, 50, 100, 1024, 2048]:
                 for periodic in [True, False]:
-                    res = torch_method(size, periodic=periodic, device=device)
+                    res = torch_method(size, *args, periodic=periodic, device=device)
                     # NB: scipy always returns a float32 result
-                    ref = torch.from_numpy(signal.get_window(name, size, fftbins=periodic))
+                    ref = torch.from_numpy(signal.get_window((name, *args), size, fftbins=periodic))
```

Comment:
I don't think the signature for `get_window()` will do what you like. Instead you can pass the torch function and the reference as a pair:

```
(torch.kaiser_window, scipy.signal.kaiser)
```